targets/hdzero.ini

```diff
@@ -24,3 +24,16 @@ lib_ignore = STM32UPDATE
 
 [env:HDZero_RX5.1_ESP32_Backpack_via_WIFI]
 extends = env:HDZero_RX5.1_ESP32_Backpack_via_UART
+
+[env:HDZero_Goggle_RX5.1_ESP32_Backpack_via_UART]
+extends = env_common_esp32, hdzero_vrx_backpack_common
+monitor_speed = 115200
+build_flags =
+	${env_common_esp32.build_flags}
+	${hdzero_vrx_backpack_common.build_flags}
+	-D PIN_LED=4
+	-D NO_AUTOBIND=1
```

Comment:
This seems to have been added to prevent the 3x power cycle bind method... any reason we don't want to allow that on the HDZ goggles?

Reply:
We were initially running into problems where the GPIO line on the goggles would be in a floating state and the backpack would randomly start in bind mode. And since we've added a command to enter bind mode from the goggle it seemed rather superfluous.

Reply:
Hmm ok.
So long as we have it documented somewhere in the wiki / user guide that the 3x power cycle works for everything EXCEPT the HDZ goggles, then I'm cool with this.

Reply:
Roger, I'll write something appropriate in the wiki.

Reply:
I really need to write a whole new section for HDZ goggles backpack, that shows all the features, for the documentation website. Flashing, wifi-mode, binding, wireless HT etc.